utils/contextutils/context.go

```diff
@@ -0,0 +1,65 @@
+// Package contextutils provides utility for adding and retrieving metadata to/from a context.
+package contextutils
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+type contextKey string
+
+const (
+	// MetadataKey is the key used to access metadata from a context with metadata.
+	MetadataKey = contextKey("viam-metadata")
+
+	// TimeRequestedMetadataKey is optional metadata in the gRPC response header that correlates
+	// to the time right before the point cloud was captured.
+	TimeRequestedMetadataKey = "viam-time-requested"
```

Comment:
these other consts need to be contextKey wrapped as well

Reply:
Oh whoops that's my bad. I assumed this const block was all context keys. will add another comment to clear up confusion